docs/iris/src/developers_guide/contributing_documentation.rst

```diff
@@ -1,49 +1,41 @@
 
-.. toctree::
-   :maxdepth: 2
-
 .. _contributing.documentation:
 
 Contributing to the documentation
-==================================
+---------------------------------
 
 Documentation is important and we encourage any improvements that can be made.
-If you believe the documentation is not clear please contribute a change to 
+If you believe the documentation is not clear please contribute a change to
 improve the documentation for all users.
 
-Any change to the Iris project whether it is a bugfix, new feature or 
-documentation update must use the :ref:`development-workflow`. 
+Any change to the Iris project whether it is a bugfix, new feature or
+documentation update must use the :ref:`development-workflow`.
 
 .. contents:: Contents:
       :local:
 
 
 Requirements
-------------
+~~~~~~~~~~~~
 
-The documentation uses specific packages that need to be present.  Please see 
+The documentation uses specific packages that need to be present.  Please see
 :ref:`installing_iris` for instructions.
 
 
 .. _contributing.documentation.building:
 
 Building
---------
-
-The build is run from the documentation directory ``iris/docs/iris/src``.  In
```

Comment:
I think it would be good to reinstate the info for what directory you need to be in.